frontend/src/components/Products.css

```diff
@@ -13,8 +15,9 @@ body {
 }
 
 .product-title {
-  font-family: 'Amatic SC';
+  font-family: 'Amatic SC', cursive;
```

Comment:
Ez a 3 sor (font-family, font-weight, font-size) elég általánosnak tűnik. Lehet jobb lenne valami általános css-be kipakolni és a megfelelő classokat rátenni az adott tag-ekre.

Reply:
Már volt róla szó, hogy a css file-okat átrendezzük hasonló okból kifolyólag. Jelenleg több ticket is css változtatásokból áll, ezért nem tudom mennyire kavarna be, ha most elkezdeném átírni file-okat?

Reply:
@taritamas89 Szerintem engedd be, ha át kell szervezni a css-eket, az legyen egy technikai refaktor task. Mit gondolsz?

Reply:
nekem jó később is ha erre lesz dedikált task